tasks/kmt.py

```diff
@@ -406,6 +406,16 @@ def full_arch(arch: ArchOrLocal) -> Arch:
     return arch
 
 
+def standardize_arch(arch: str) -> Arch:
```

Comment:
We have another function which does similar things called `full_arch`. Can we remove that and replaces it's usage with this?

Reply:
Changed `full_arch` so that it converts any arch-looking string to a full arch